recipes/hpv-kite/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "hpv-kite" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpdcj/HPV-KITE/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e928ceb741b4b1fd3ade33e6204530e5998c4b5091f4a9d7f2e094bdef897ec7
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - openjdk >=17,<22
```

```suggestion
    - openjdk >=17,<22
    - sed
```
For good measure let's make sure we are using GNU `sed` - BSD `sed` has issues with more advanced `sed` operations.

Also, I think we should just pin to `openjdk =21` here because that is newest LTS version your software supports.